openfisca_core/legislation.xsd

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+
+    <xs:element name="NODE" type="nodeType"/>
+
+    <xs:group name="piecewiseConstant">
```

Comment:
Ok, I get it. So for instance this is a `piecewiseConstant `:
```xml
      <END deb="2012-01-01" />
      <VALUE deb="2009-01-01" valeur="0.6" />
      <VALUE deb="2008-01-01" valeur="0.5" />
```
Not sure this is the best way to describe it in the context of OpenFisca. Some suggestions : 
- `valuesHistory`
- `datedValues`
- `parameterHistory`